### PR TITLE
Add Main Page

### LINKED ISSUE #5: [4.0pt] - Add Main Page
- Create the component and set a route in the App
- List all the experiences available, displaying images and descriptions for each one
- When the user clicks one experience the app has to redirect to the show page for that experience.
- Add styles.

### PR DESCRIPTION
**Pull request by Dachrono**
# 🎉 🎈 In this *Milestone* 🙋‍♂️ 🚩 ‼ :

**Hi there** 👋

- Follow the best GitHub flow. ✅
- Follow our list of best practices for Ruby. ✅
- Document in a professional way. ✅
- No linters errors. ✅
- Create Main page. ✅
- List of all experiences available. ✅
- Create a link for redirection to show page for every experience. ✅
- Add requirement style. ✅

# To the code reviewer 👨‍💻
I really thank 🙇‍♂️ you, dear code reviewer 👨‍🎨 for dedicating your time ⌚ to check the PR 🧾 .

I kindly ask code reviewer 🤵 that if there is any issue in this project, please do list 📃 them in a descriptive 💡 manner and give your best suggestions if needed.

If you think some significant issues need to be changed ♻️ please kindly contact 📞 me through Zoom or Slack, I am available on Microverse UTC-6 time zone.

### CODE REVIEW

File: src/Components/MainPage.jsx [hunk 1]
```diff
@@ -1,22 +1,44 @@
...
-      <p>
-        Esto es la página principal para
-        {storageData.user.name}
-      </p>
+      <h1>{`Welcome ${DataUser.user.name}`}</h1>
+      <p className="secondtitle">at the best experiences in the world</p>
+      <swiper-container slides-per-view="3" navigation="true" class="swiper-container">
+        {DataExperiences.experiences.map((item) => (
+          <swiper-slide key={item.id}>
+            <Link className="cardlink" to={`./Register/${item.id}`}>
```
Comment (NitBravoA92) [inline src/Components/MainPage.jsx:24]: @Dachrono I suggest changing the path to the Experience Details page. A possible path would be: `/experiences/${item.id}`

**Review by NitBravoA92 — APPROVED** [review]
# Status: Approved :green_circle:

### Hi @Dachrono  

EXCELENTE JOB!! I can notice your effort in this new implementation. It is complete so it's time to merge it 💯 🥳.

congrats

## Highlights :1st_place_medal:

- Beautiful UI design
- Good use of `useSelector()` hook to access the store
- Good job using a modern slider library: `Swiper.js`
- Good job displaying the Experience information on each card

# OPTIONAL CHANGES
- I have left a set of key suggestions that can help improve this implementation. Please consider implementing them.

_All the changes with optional tags are not crucial enough to prevent you from getting the approval. Still, I would highly recommend that you consider these changes._

Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

File: src/Components/MainPage.jsx [hunk 2]
```diff
@@ -1,22 +1,44 @@
...
+              <div className="card">
+                <h2>{item.name}</h2>
+                <img src={item.image} alt="fotito" />
```
[the remark this answers is not in the document]
Comment (Dachrono) [reply src/Components/MainPage.jsx]: Hello Carmen, tky for your time and your comments I appreciate it!

File: src/Components/MainPage.jsx (hunk 1 above)
Comment (Dachrono) [reply src/Components/MainPage.jsx]: Hello Carmen, tky for your time and your comments I appreciate it!

**Pull request merged**